__mocks__/inquirer/prompt.mock.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+'use strict';
+const Rx = require('rx');
+const inquirer = jest.genMockFromModule('inquirer');
+
+function prompt(questions, config, testAnswer) {
+ 	return questions.reduce(function (newOpts, ans) {
+		return attachAnswers(newOpts, testAnswer);
+	}, Object.assign(config, require('../../lib/utils/initial-config')))
+  	.map(newOpts => {
+		return checkEmptyAnswers(newOpts);
```

Comment:
No need for the `return` here. `.map(newOpts => checkEmptyAnswers(newOpts))` or even `.map(checkEmptyAnswers)`